static/js/tools/map/stat_var_chooser.tsx

```diff
@@ -108,7 +108,12 @@ export function StatVarChooser(props: StatVarChooserProps): JSX.Element {
       samplePlaces={samplePlaces}
       deselectSVs={deselectSVs}
       selectedSVs={selectedSVs}
-      selectSV={(svDcid) => selectStatVar(statVar, display, placeInfo, svDcid)}
+      selectSV={(svDcid) => {
+        selectStatVar(statVar, display, placeInfo, svDcid);
+        window &&
```

Comment:
I noticed this same gtag added as part of the selectSV props for all of our tools. Maybe we can just add the gtag to the StatVarHierarchy component where the props.selectSV is used: https://github.com/datacommonsorg/website/blob/78bc010552434c8d41f0e3d92c1638b24c43da58/static/js/stat_var_hierarchy/stat_var_hierarchy.tsx#L298

Reply:
Done